index.js

```diff
@@ -0,0 +1,88 @@
+const q = require('daskeyboard-applet');
+const request = require('request-promise');
+const logger = q.logger; 
+
+// Check the price of a selected flight every minute in order to buy it at the best price
+// Compare the current price to the old price stored one minute ago
+
+class FlyMe extends q.DesktopApp {
+	
+    constructor() {
+        super();
+        // every minute
+        this.pollingInterval = 60*1000; // ms
+		logger.info("Get the cheapest flight price!");
```

Comment:
Indent line correctly. You should use a linter and a formatter.

+ More specific message like "FlightPriceWatcher starting: Get the cheapest flight price!"